include/dali/core/math_util.h

```diff
@@ -167,6 +167,25 @@ constexpr double rad2deg(double rad) {
   return rad * r2d;
 }
 
+/// @brief Calculates normalized sinc i.e. `sin(pi * x) / (pi * x)`
+DALI_HOST_DEV DALI_FORCEINLINE
```

Comment:
unnecessarily duplicated code. You can templatize this and replace `1.0` and `1.0f` by `T(1)`

Reply:
The threshold for transition to approximation is different.